test/manifestdata.py

```diff
@@ -48,7 +50,9 @@
         "label": "sam",
         "manifest": true
     },
-    "uuid": "ba5ac769-207e-421c-bfd2-a23c767114af"
+    "uuid": "ba5ac769-207e-421c-bfd2-a23c767114af",
+    "urlApi":"https://subscription.rhn.stage.redhat.com/subscription/consumers/"
```

Comment:
There is missing comma at the end of the line. There should be space between `"urlApi":` and "https://...`. The space is also missing at following line.

Reply:
ups, too much copy paste, fixed